packages/realm-server/server.ts

```diff
@@ -336,15 +338,25 @@ export class RealmServer {
       ...(backgroundURL ? { backgroundURL } : {}),
     });
     if (this.seedPath) {
-      let ignoreList = IGNORE_SEED_FILES.map((file) =>
-        join(this.seedPath!.replace(/\/$/, ''), file),
-      );
-      copySync(this.seedPath, realmPath, {
-        filter: (src, _dest) => {
-          return !ignoreList.includes(src);
-        },
-      });
-      this.log.debug(`seed files for new realm ${url} copied to ${realmPath}`);
+      if (copyFromSeedRealm) {
+        let ignoreList = IGNORE_SEED_FILES.map((file) =>
+          join(this.seedPath!.replace(/\/$/, ''), file),
+        );
+
+        copySync(this.seedPath, realmPath, {
+          filter: (src, _dest) => {
+            return !ignoreList.includes(src);
+          },
+        });
+        this.log.debug(
+          `seed files for new realm ${url} copied to ${realmPath}`,
+        );
+      } else {
+        copySync(
+          join(this.seedPath!.replace(/\/$/, ''), 'index.json'),
+          join(realmPath!.replace(/\/$/, ''), 'index.json'),
+        );
+      }
     }
 
     let realm = new Realm(
```

Comment:
you are going to have to also make sure to not set the `copiedFromRealm` value in the `Realm` constructor options below (after you merge from upstream).

Reply:
Updated it here https://github.com/cardstack/boxel/pull/2077/files#diff-14c49ddb85659961b6177a036fc15d7166291f81169c9a1dd0dfead082ef68e5R385